plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -3963,4 +3965,81 @@ public VirtualMachineMO cloneVMFromTemplate(VmwareHypervisorHost hyperHost, Stri
             throw new CloudRuntimeException(msg, e);
         }
     }
+
+    @Override
+    public Answer SyncVolumePath(SyncVolumePathCommand cmd) {
+        DiskTO disk = cmd.getDisk();
+        VolumeObjectTO volumeTO = (VolumeObjectTO)disk.getData();
+        DataStoreTO primaryStore = volumeTO.getDataStore();
+        String volumePath = volumeTO.getPath();
+        String vmName = cmd.getVmName();
```

Comment:
VolumeObjectTO already holds the VM name, and you can re-use the same

Reply:
Updated syncvolumepath command by removing vmname parameter.